dependencies.gradle

```diff
@@ -62,8 +62,8 @@ def VERSIONS = [
         'org.hsqldb:hsqldb:2.5.+', // 2.6.0 requires JDK 11.
         'org.jooq:jooq:3.14.+',
         'org.jsr107.ri:cache-ri-impl:1.0.0',
-        'org.junit.jupiter:junit-jupiter:5.8.+',
-        'org.junit.platform:junit-platform-launcher:1.8.+',
+        'org.junit.jupiter:junit-jupiter:latest.release',
+        'org.junit.platform:junit-platform-launcher:latest.release',
```

Comment:
The reason I didn't use `latest.release` for JUnit dependencies is because JUnit releases milestone versions to Maven Central. So depending on the timing of our minor releases, we may end up pinned to a milestone release for that minor version. So avoiding the milestones was the intention.

Reply:
Makes sense, I reverted it in https://github.com/micrometer-metrics/micrometer/commit/2fa87beeffc698ec63fd24866d4f8f4517e78680

Reply:
Sorry, I didn't know the intention. I created https://github.com/micrometer-metrics/micrometer/pull/2890 to add a note not to change it accidentally in the future again.